timelock-agent/src/main/java/com/palantir/timelock/paxos/TimeLockAgent.java

```diff
@@ -588,6 +588,7 @@ public long getLatestSchemaVersion() {
     // No runtime configuration at the moment.
     private void registerPaxosResource() {
         paxosResources.resourcesForRegistration().forEach(registrar);
+        paxosResources.undertowServices().forEach(service -> undertowRegistrar.ifPresent(reg -> reg.accept(service)));
```

Comment:
This is pretty sketchy, worth discussing a path forward.

At the moment it's fine if `undertowRegistrar` isn't set because we register both UndertowService and jersey components for the same endpoint.

Reply:
Hmm. Yeah I think the only users of this class should be
- internal wrapper around timelock
- some classes we use here for testing.

So agree it looks weird, but there's no immediate cause for alarm